### ISSUE TITLE
Module broken after Ebean 13.9.1 upgrade

### ISSUE THREAD

**Issue by vladimirfx** [ebean-orm/ebean-spring-txn#28]
Method `JdbcTransaction.isActive()` becomes final witch is broken any external transaction support:

```log
Caused by: java.lang.IncompatibleClassChangeError: class io.ebean.spring.txn.SpringJdbcTransaction overrides final method io.ebeaninternal.server.transaction.JdbcTransaction.isActive()Z
    	at io.ebean.spring.txn.SpringJdbcTransactionManager.getCurrentTransaction(SpringJdbcTransactionManager.java:85) ~[ebean-spring-txn-13.0.0.jar:?]
    	at io.ebeaninternal.server.transaction.ExternalTransactionScopeManager.active(ExternalTransactionScopeManager.java:27) ~[ebean-core-13.9.1.jar:13.9.1]
    	at io.ebeaninternal.server.transaction.TransactionManager.active(TransactionManager.java:184) ~[ebean-core-13.9.1.jar:13.9.1]
    	at io.ebeaninternal.server.core.DefaultServer.currentServerTransaction(DefaultServer.java:724) ~[ebean-core-13.9.1.jar:13.9.1]
    	at io.ebeaninternal.server.core.DefaultServer.buildQueryRequest(DefaultServer.java:960) ~[ebean-core-13.9.1.jar:13.9.1]
    	at io.ebeaninternal.server.core.DefaultServer.buildQueryRequest(DefaultServer.java:955) ~[ebean-core-13.9.1.jar:13.9.1]
    	at io.ebeaninternal.server.core.DefaultServer.createQueryRequest(DefaultServer.java:946) ~[ebean-core-13.9.1.jar:13.9.1]
    	at io.ebeaninternal.server.core.DefaultServer.findIdsWithCopy(DefaultServer.java:1214) ~[ebean-core-13.9.1.jar:13.9.1]
    	at io.ebeaninternal.server.core.DefaultServer.findIds(DefaultServer.java:1208) ~[ebean-core-13.9.1.jar:13.9.1]
    	at io.ebeaninternal.server.querydefn.DefaultOrmQuery.findIds(DefaultOrmQuery.java:1425) ~[ebean-core-13.9.1.jar:13.9.1]
```

**Comment by rbygrave** [comment]
This is fixed in ebean 13.9.2

---

Note: Just bumping up ebean-spring-txn to ebean 13.9.2 and the `SpringTxnListener.flush()` needed updating to use underlying transaction.flush() from transaction.flushBatch() as per https://github.com/ebean-orm/ebean-spring-txn/pull/29 ... so I have made that change and released ebean-spring-txn 13.9.0 with that change.

**Comment by vladimirfx** [comment]
The issue was fixed with Ebean 13.9.2. Waiting for transaction.flush fix

**Comment by rbygrave** [comment]
Cool !!!
